packages/zoe/test/unitTests/zcf/registerFeeMintContract.js

```diff
@@ -26,7 +27,7 @@ const start = async (zcf, privateArgs) => {
     zcfSeat,
   );
 
-  const creatorFacet = harden({
+  const creatorFacet = Far('mint creator facet', {
```

Comment:
Why did this previously work with no `Far` declaration?

It is good now, so no change suggested. But I do not understand why things used to work.